.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--
+Thank you for sending the PR! We appreciate you spending the time to work on these changes.
+
+Help us understand your motivation by explaining why you decided to make this change.
+You can learn more about contributing to Mypy here: https://github.com/python/mypy/blob/master/CONTRIBUTING.md
+
+Happy Contributing!
+
+-->
+
+### Have you read the [Contributing Guidelines](https://github.com/python/mypy/blob/master/CONTRIBUTING.md)?
+
+(Write your answer here.)
+
+### Description
+
+(Write your answer here.)
+
+## Test Plan
+
+(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)
```

Comment:
I don't think so. We have automated tests. Every PR that changes code should have tests (unless it's a pure refactoring that doesn't change any external behavior -- but we're not so keen on those). So we should provide specific, actionable instructions, e.g.

- If this is a documentation change, rebuild the docs (_link to instructions_) and review the changed pages for markup errors.
- If this is a code change, include new tests (_link to the testing docs_). Be sure to run the tests locally and fix any errors before submitting the PR (_more instructions_).
- If this change cannot be tested by the CI, please explain how to verify it manually.